src/components/NavBar.jsx

```diff
@@ -0,0 +1,22 @@
+import { Link } from "react-router-dom";
+
+export default function NavBar() {
+  return (
+    <nav className="p-4 bg-slate-600">
+      <ul className="flex flex-col items-center justify-center gap-2 sm:flex-row sm:gap-20">
```

Comment:
If you use `flex mx-auto` (if I remember correctly) you can greatly reduce this class name without having to worry about the media queries possibly.

Reply:
Looks like the ```mx-auto``` is not applying the margin when changing viewports. I will look into this more.